src/ui/pages/SwapForm/SlippageSettings/SlippageSettings.tsx

```diff
@@ -117,59 +186,84 @@ export function SlippageSettings({
-            setIsCustomValue(true);
+      <VStack gap={16} style={{ alignContent: 'start', flexGrow: 1 }}>
+        <div
+          style={{
+            display: 'grid',
+            gridTemplateColumns: `repeat(${SLIPPAGE_OPTIONS.length}, 1fr) 1fr`,
+            gap: 8,
           }}
-          onChange={(event) => setPercentValue(event.currentTarget.value)}
-          pattern={FLOAT_INPUT_PATTERN}
-          required={isCustomValue}
-        />
-      </div>
-      <Spacer height={16} />
-      <SlippageWarning percentValue={percentValue} />
-      <Spacer height={32} />
+        >
+          {SLIPPAGE_OPTIONS.map((value) => (
+            <Radio
+              key={value}
+              name="slippage"
+              value={value}
+              checked={!isCustomValue && value === percentValue}
+              onChange={() => {
+                setPercentValue(value);
+                setIsCustomValue(false);
+              }}
+              onFocus={() => {
+                setIsCustomValue(false);
+              }}
+              required={!isCustomValue}
+            >
+              <UIText kind="body/accent">{`${value}%`}</UIText>
+            </Radio>
+          ))}
+          <div style={{ position: 'relative' }}>
+            <PercentWidth onValue={setPercentCharWidth} />
+            {persentCharWidth ? (
+              <CustomValueOverlay
+                value={isCustomValue ? percentValue : null}
+                isOptimal={isOptimal}
+              />
+            ) : null}
+            <Input
+              name="customSlippage"
+              value={isCustomValue ? percentValue : ''}
+              onFocus={() => {
+                if (!isCustomValue) {
+                  setPercentValue('');
+                }
+                setIsCustomValue(true);
+              }}
+              style={{
+                textAlign: 'center',
+                fontWeight: 500,
+                border: isOptimal ? undefined : '1px solid var(--notice-500)',
+                color: isOptimal ? undefined : 'var(--notice-500)',
+                paddingRight: persentCharWidth + 12, // `%` width + input default padding,
+              }}
+              onChange={(event) => {
+                setPercentValue(event.currentTarget.value);
+              }}
+              pattern={FLOAT_INPUT_PATTERN}
+              required={isCustomValue}
+            />
+          </div>
+        </div>
+        <SlippageWarning percentValue={percentValue} />
+        <UIText kind="body/regular" color="var(--neutral-500)">
+          Your transaction will fail if the price changes more than the
+          slippage.
+        </UIText>
+      </VStack>
       <div style={{ display: 'grid', gap: 8, gridTemplateColumns: '1fr 1fr' }}>
         <Button
           kind="neutral"
           type="button"
           onClick={() => {
             setIsCustomValue(false);
-            setPercentValue(SLIPPAGE_OPTIONS[0]);
+            setPercentValue(SLIPPAGE_OPTIONS[1]);
```

Comment:
I think it's fine to assume the first element as a default, but because now it's the second, perhaps it's better to extract a constant near SLIPPAGE_OPTIONS declaration:
```
const DEFAULT_SLIPPAGE_VALUE = SLIPPAGE_OPTIONS[1]
```